components/aws/sagemaker/batch_transform/README.md

```diff
@@ -0,0 +1,98 @@
+# SageMaker Batch Transform Kubeflow Pipeline component
+## Summary
```

Comment:
keep summary as big header too

`#`.

Reply:
We want an H2 header for `Summary` as it is not the name of the page.